tests/test_pipeline.py

```
@@ -152,120 +277,132 @@ def test_registered_intent_yielded(self):
-        # Model has 3 classes; only 2 are registered. Raw probs: [0.6, 0.2, 0.2]
-        _setup_model(p, ["skill_a:a.intent", "skill_b:b.intent", "skill_c:c.intent"],
-                     [0.6, 0.2, 0.2])
-        results = list(p._match("test"))
-        total = sum(prob for _, _, prob in results)
-        self.assertAlmostEqual(total, 1.0, places=6)
+class TestMatchPrototype(unittest.TestCase):
+    def _make_store(self, label_embeddings: dict):
+        """label_embeddings: {label: embedding_vector}"""
+        from ovos_m2v_pipeline import PrototypeIntentStore
+        labels = list(label_embeddings.keys())
+        embs = np.array([label_embeddings[l] for l in labels], dtype=np.float32)
```

Ruff (0.15.2)

[error] Ambiguous variable name: `l`

(E741)